src/app/pages/HomePage/index.tsx

```diff
@@ -85,9 +88,12 @@ export const HomePage: FC = () => {
         <LogotypeBox>
           <Logotype showText />
         </LogotypeBox>
-        <SearchInputBox>
-          <Search onFocusChange={onFocusChange} />
-        </SearchInputBox>
+        <Box sx={{ position: 'relative', zIndex: 2 }}>
```

Comment:
I don't like that `position: relative` is leaking out of OfflineIndicator component.

This can be avoided with:
```tsx
const StyledBox = styled(Box)(() => ({
  display: 'flex',
  justifyContent: 'center',
}))

const StyledAlert = styled(Alert)(() => ({
  position: 'absolute',
  marginTop: '10px',
}))

export const OfflineIndicator: FC = () => {
  const { t } = useTranslation()

  return (
    <StyledBox>
      <StyledAlert severity="error">{t('home.apiOffline')}</StyledAlert>
    </StyledBox>
  )
}
```
```tsx
        <Box sx={{ zIndex: 2 }}>
          <SearchInputBox>
            <Search disabled={isApiOffline} onFocusChange={onFocusChange} />
          </SearchInputBox>
          {isApiOffline && <OfflineIndicator />}
        </Box>
```

But I would also move some layouting out of the component:
```tsx
const StyledAlert = styled(Alert)(() => ({
  position: 'absolute',
  marginTop: '10px',
}))

export const OfflineIndicator: FC = () => {
  const { t } = useTranslation()
  return <StyledAlert severity="error">{t('home.apiOffline')}</StyledAlert>
}
```

```tsx
        <Box sx={{ zIndex: 2 }}>
          <SearchInputBox>
            <Search disabled={isApiOffline} onFocusChange={onFocusChange} />
          </SearchInputBox>
          {isApiOffline && (
            <Box sx={{ display: 'flex', justifyContent: 'center' }}>
              <OfflineIndicator />
            </Box>
          )}
        </Box>
```